src/Microsoft.VisualStudio.ProjectSystem.CSharp.VS/VSPackage.resx

```diff
@@ -117,10 +117,20 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="#1" xml:space="preserve">
+  <data name="1" xml:space="preserve">
```

Comment:
The ProjectTypeRegistration attribute mentions # 1 - doesn't it have to be called that for it to find the resource?